servant-foreign/servant-foreign.cabal

```diff
@@ -0,0 +1,27 @@
+name:                servant-foreign
+version:             0.5.0.0
```

Comment:
I usually prefer `0.5` and then only use the next two numbers when doing minor updates to the library. But that's just my opinion.